internal/dao/tests/main_test.go

```diff
@@ -27,7 +27,7 @@ func teardown() {
 }
 
 func initEnvironment() {
-	config.Initialize("config/test.env")
+	config.Initialize("../../../config/test.env")
```

Comment:
I don't know if ppl had their config in this folder, but this makes bit more sense to me :thinking:

Reply:
I do not want to start from weird directory, I want to run `make integration-tests` from the root directory of the project. I think it's sensible. Without this lookup it doesn't work. It's not `CWD` in this case, it's the main fail location - which is this file, so it is relative path to this file.

My aim was to make it work with config in the main `config/` directory, but I might have done something wrong :)

Reply:
I thought it will be no brainer, but if it needs to be discussed, lets take it to another PR: #337.